static/js/components/VideoPlayer.js

```diff
@@ -129,38 +145,66 @@ export default class VideoPlayer extends React.Component {
     this.videoNode.style.left = `${left}px`;
     this.videoNode.style.top = `${top}px`;
     // $FlowFixMe prop does not have to be a number
-    this.videoNode.style[prop] = 'scale(2)';
+    this.videoNode.style[prop] = "scale(2)";
     this.configureCameras();
 
   }
 
   toggleFullscreen() {
-    if (isFullscreen()) {
+    let fullscreen = isFullscreen();
+    if (fullscreen) {
       // $FlowFixMe
       document[FULLSCREEN_API.exitFullscreen]();
     } else {
       // $FlowFixMe videoContainer.parentElement is not going to be null
       this.videoContainer.parentElement[FULLSCREEN_API.requestFullscreen]();
     }
+    this.player.el_.dispatchEvent(new Event(`fullscreen ${fullscreen ? "off" : "on"}`));
+  }
+
+  analyzeEvent(action: string, label: string) {
+    let percentTracked = this.percentTracked;
+    this.player.on(action, function() {
+      if (action === "timeupdate") {
+        // Track percentage played in increments of 10
+        const currentTime = Math.round(this.currentTime());
+        const duration = Math.round(this.duration());
+        let percentPlayed = Math.round(currentTime / duration * 100);
+        _.range(10,100,10).forEach((percent:number) => {
+          if (percentPlayed >= percent && !percentTracked.includes(percent)) {
```

Comment:
idk if I'm reading this right, but it seems like what we want to do is find the closest multiple of 10 percentage value that is `<=` to our current percentage, right?

if that's right I think we can avoid iterating over this `_.range` call by doing this:

```js
const percentRoundTen = percentPlayed - (percentPlayed % 10)
```